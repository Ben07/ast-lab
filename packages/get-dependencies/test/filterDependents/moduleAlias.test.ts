// @ts-ignore
import { enumerateArgsTestFunction, configArgs } from 'lazy-jest';
import { filterDependents } from './utils';

enumerateArgsTestFunction(
  filterDependents.bind(null, 'module-alias'),
  configArgs()
  .arg('sources', [
    ['./a.js', 'unrelated-folder.js'],
    ['./a.js', 'unrelated-name.js'],
  ])
  .arg('targets', [
    { './a': ['default'] }
  ])
  .arg('options', [{
    alias: {
      'alias-folder': './',
      'alias-a': './a'
    }
  }]),
  'module alias'
);