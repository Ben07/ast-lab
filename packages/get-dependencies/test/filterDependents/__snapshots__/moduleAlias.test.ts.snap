// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`module alias - (["sources", "targets", "options"]) - [["./a.js","unrelated-folder.js"],{"./a":["default"]},{}] 1`] = `
Array [
  "/module-alias/a.js",
  "/module-alias/unrelated-folder.js",
]
`;

exports[`module alias - (["sources", "targets", "options"]) - [["./a.js","unrelated-name.js"],{"./a":["default"]},{}] 1`] = `
Array [
  "/module-alias/a.js",
  "/module-alias/unrelated-name.js",
]
`;
